apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -420,6 +397,44 @@ const EventTypePage = (props: EventTypeSetupProps) => {
     formState: { isDirty: isFormDirty, dirtyFields },
   } = formMethods;
 
+  useEffect(() => {
+    const handleRouteChange = (url: string) => {
+      const paths = url.split("/");
+
+      // We don't need to show the modal if the user is assigning all team members as they are already assigned
+      if (formMethods.getValues("assignAllTeamMembers") === true) {
+        return;
+      }
```

Comment:
If all team members toggle is checked we don't need to worry about showing the modal as there is users assigned but in a different way